### PR TITLE
Add some learning reference in ID

### PR DESCRIPTION
**Pull request by lenfaire**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by LuigiImVector** [comment]
To access these resources is required an account so you need to add this `(account *requested*)` to the end of each string.

Maybe you could also add the name of the course authors (example: `* [Flutter Membangun Website](https://buildwithangga.com/kelas/flutter-membangun-website-sederhana?thumbnail=nk4neM2UyG.110&main_leads=browse) - Rifqi Eka, BuildWithAngga (account *requested*)`

File: courses/free-courses-id.md [hunk 1]
```diff
@@ -95,6 +95,7 @@
 
 ### Flutter
 
+* [Flutter Membangun Website](https://buildwithangga.com/kelas/flutter-membangun-website-sederhana?thumbnail=nk4neM2UyG.110&main_leads=browse) - BuildWithAngga
```
Comment (davorpa) [inline courses/free-courses-id.md:98]: Creator: Rifqi Eka
Cleaned link: https://buildwithangga.com/kelas/flutter-membangun-website-sederhana

Use this snippet to also merge pull #5589 and as main fix:
```markdown
* [Flutter for Designer: Design to Code](https://buildwithangga.com/kelas/flutter-for-designer-design-to-code) - Rifqi Eka *(BuildWithAngga account and email address required)*
* [Flutter Membangun Website](https://buildwithangga.com/kelas/flutter-membangun-website-sederhana) - Rifqi Eka *(BuildWithAngga account and email address required)*
```

File: courses/free-courses-id.md [hunk 2]
```diff
@@ -109,6 +110,7 @@
 
 * [Belajar Go-Lang untuk Pemula](https://www.youtube.com/playlist?list=PL-CtdCApEFH_t5_dtCQZgWJqWF45WRgZw) - Programmer Zaman Now
 * [Belajar Golang Basic Bahasa Indonesia](https://www.youtube.com/playlist?list=PLCZlgfAG0GXDztO-BFc9R5afhP26Dhsgm) - Sekolah Koding
+* [Golang Fundamental](https://buildwithangga.com/kelas/golang-fundamental?thumbnail=nk4neM2UyG.54&main_leads=browse) - BuildWithAngga
```
Comment (davorpa) [inline courses/free-courses-id.md:113]: Creator: Agung Setiawan
Right link: https://buildwithangga.com/kelas/golang-fundamental

Use this snippet as fix:
```markdown
* [Golang Fundamental](https://buildwithangga.com/kelas/golang-fundamental) - Agung Setiawan *(BuildWithAngga account and email address required)*
```

File: courses/free-courses-id.md [hunk 3]
```diff
@@ -117,6 +119,7 @@
 * [CSS Dasar](https://www.youtube.com/playlist?list=PLFIM0718LjIUBrbm6Gdh6k7ZUvPIAZm7p) - Web Programming UNPAS
 * [CSS Grid](https://www.youtube.com/playlist?list=PLFIM0718LjIXmbwX0dEsoRVX-PC16vmuw) - Web Programming UNPAS
 * [CSS Layouting](https://www.youtube.com/playlist?list=PLFIM0718LjIVCmrSWbZPKCccCkfFw-Naa) - Web Programming UNPAS
+* [CSS Tailwind](https://buildwithangga.com/kelas/css-tailwind-web-design?thumbnail=nk4neM2UyG.46&main_leads=browse) - BuildWithAngga
```
Comment (davorpa) [inline courses/free-courses-id.md:122]: Creator: Yein Narayana
Right title: CSS Tailwind Web Design
Right link: https://buildwithangga.com/kelas/css-tailwind-web-design

Use this snippet as fix:
```markdown
* [CSS Tailwind Web Design](https://buildwithangga.com/kelas/css-tailwind-web-design) - Yein Narayana *(BuildWithAngga account and email address required)*
```

File: courses/free-courses-id.md [hunk 4]
```diff
@@ -153,6 +156,7 @@
 
 #### Node
 
+* [Node.js Dasar](https://buildwithangga.com/kelas/node-javascript-dasar?thumbnail=nk4neM2UyG.36&main_leads=browse) - BuildWithAngga
```
Comment (davorpa) [inline courses/free-courses-id.md:159]: Creator: Elfin Sanjaya
Right title: Node JavaScript Dasar
Right link: https://buildwithangga.com/kelas/node-javascript-dasar

Use this snippet as fix:
```markdown
* [Node JavaScript Dasar](https://buildwithangga.com/kelas/node-javascript-dasar) - Elfin Sanjaya *(BuildWithAngga account and email address required)*
```

File: courses/free-courses-id.md [hunk 5]
```diff
@@ -202,6 +206,7 @@
 
 ### Microservices
 
+* [Microservice Architecture Web Development](https://buildwithangga.com/kelas/microservice-architecture-web-development?thumbnail=nk4neM2UyG.49&main_leads=browse) - BuildWithAngga
```
Comment (davorpa) [inline courses/free-courses-id.md:209]: Creator: Widada
Right link: https://buildwithangga.com/kelas/microservice-architecture-web-development

Use this snippet as fix:
```markdown
* [Microservice Architecture Web Development](https://buildwithangga.com/kelas/microservice-architecture-web-development) - Widada *(BuildWithAngga account and email address required)*
```

**Review by davorpa — CHANGES_REQUESTED** [review]
Following CONTRIBUTING format guidelines.
- Links can't have tracking parameters, so clean it.
- Give attributions to the creator/mentor/platform
- For resource name, use the title as is published in the website
- Add notes about platform registering: email and phone is required.

Please, apply following snippets to solve it.

**Comment by eshellman** [comment]
Nice teamwork. 
Thanks, and welcome to free-programming-books!

**Comment by lenfaire** [comment]
Thank you sir. Glad being contributor on this project.

**Pull request merged**